docs/volume/volume-snapshots.md

```diff
@@ -8,33 +8,48 @@ keywords:
 Description: Take a snapshot for a volume from the Volume page.
 ---
 
-## Create Volume Snapshots
+A volume snapshot represents a snapshot of a volume on a storage system. After creating a volume, you can create a volume snapshot and restore a volume to the snapshot's state. With volume snapshots, you can easily copy or restore a volume's configuration.
 
-:::note
+## Create volume snapshots
 
-A recurring snapshot is currently not supported and is tracked via [harvester/harvester#572](https://github.com/harvester/harvester/issues/572)
+You can create a volume snapshot from an existing volume by following these steps:
 
-:::
+1. Go to the **Volumes** page.
+1. Choose the volume that you want to take a snapshot of and select **⋮ > Take Snapshot**.
+
+ ![create-volume-snapshot-1](/img/v1.2/volume/create-volume-snapshot-1.png)
+1. Enter a **Name** for the snapshot.
+
+ ![create-volume-snapshot-2](/img/v1.2/volume/create-volume-snapshot-2.png)
+1. Select  **Create** to finish creating a new volume snapshot.
 
-After creating a volume, you can create volume snapshots by following the steps below:
+1. You can view all VM snapshots and check the status of this operation from the **Backup & Snapshot > VM Snapshots** page. When the **ReadyToUse** status becomes **true**, the snapshot is created successfully.
```

Comment:
This description is about `VM Snapshots` but this page is about `Volume Snapshots`. The user cannot see volume snapshots from the `VM Snapshots` page.